netplan/libnetplan.py

```diff
@@ -28,6 +28,18 @@ class LibNetplanException(Exception):
     pass
 
 
+class LibNetplanPermissionException(Exception):
```

Comment:
Those classes should probably inherit from a common `NetplanException` base class, as suggested by Simon.